bridge_ui/src/contexts/EthereumProviderContext.tsx

```diff
@@ -219,8 +221,9 @@ export const EthereumProviderProvider = ({
         EthereumProvider.init({
           projectId: WALLET_CONNECT_PROJECT_ID,
           showQrModal: false,
-          chains: [getEvmChainId(wormholeChainId) as number],
-          rpcMap: EVM_RPC_MAP
+          chains: [getEvmChainId(CHAIN_ID_ETH) as number, getEvmChainId(CHAIN_ID_BSC) as number],
```

Comment:
In the latest WalletConnect, switching chains does not require user approval but instead requires specifying all chain ids when initializing.